README.md

```diff
@@ -3,4 +3,24 @@ An archive of research
 
 A project from OneVariable 
 
-James Munns & Amanda Majorowicz
\ No newline at end of file
+James Munns & Amanda Majorowicz
+# Project Page Organization
```

Comment:
```suggestion

# Project Page Organization

```

Personal preference, I like having a blank line before and after any section headers like this.